gluten-core/src/main/scala/io/glutenproject/execution/WholeStageTransformerExec.scala

```diff
@@ -149,7 +149,7 @@ case class WholeStageTransformerExec(child: SparkPlan)(val transformStageId: Int
       maxFields: Int,
       printNodeId: Boolean,
       indent: Int = 0): Unit = {
-    val prefix = if (printNodeId) "* " else s"*($transformStageId) "
+    val prefix = if (printNodeId) "^ " else s"^($transformStageId) "
```

Comment:
what's the different between `*` and `^` ?

Reply:
Using ^ , so we can distinguish plan between whole stage codegen and whole stage transformer.
In spark plan , * usually means the plan node is codegened, same ID means they are in the same whole stage codegen.
We can keep the ID part, but use a ^ or other symbols instead.